web-tester/pom.xml

```diff
@@ -0,0 +1,126 @@
+  <packaging>jar</packaging>
+  
+  <name>web-tester</name>
+  <description>API that simplifies web testing with ivy projects.</description>
+  <url>https://github.com/ivy-supplements/web-tester/</url>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>IvyTeam</name>
+      <email>support@ivyteam.ch</email>
+      <organization>AXON Ivy AG</organization>
+      <organizationUrl>https://www.axonivy.com</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git@github.com:ivy-supplements/web-tester.git</connection>
+    <developerConnection>scm:git:git@github.com:ivy-supplements/web-tester.git</developerConnection>
+    <url>git@github.com:ivy-supplements/web-tester.git</url>
+  </scm>
+
+  <organization>
+    <name>The open source community of Axon.ivy</name>
+    <url>https://github.com/ivy-supplements</url>
+  </organization>
+  
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <distributionManagement>
+    <snapshotRepository>
+      <id>ossrh</id>
+      <name>Maven Central Snapshots Repo</name>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+  </distributionManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.axonivy.ivy.webtest</groupId>
+      <artifactId>primeui-tester</artifactId>
+      <version>7.0.0-SNAPSHOT</version>
+    </dependency>
+  </dependencies>
+  
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.1</version>
+        <configuration>
+          <source>11</source>
+          <target>11</target>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-source-plugin</artifactId>
+        <version>3.2.0</version>
+        <executions>
+          <execution>
+            <id>attach-sources</id>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <version>3.1.1</version>
+        <configuration>
+          <javadocExecutable>${java.home}/bin/javadoc</javadocExecutable>
+        </configuration>
+        <executions>
+          <execution>
+            <id>attach-javadocs</id>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-gpg-plugin</artifactId>
+        <version>1.5</version>
```

Comment:
now large parts of these new POMs files are duplicated:
- lets solve it in a common parent pom
- but use the maven flatten approach to still deliver a simple easy to use POM without hierarchies.